cli50

```diff
@@ -17,17 +17,10 @@ if sys.version_info < (3, 0):
     input = raw_input
 
 # Get version
-# https://stackoverflow.com/a/17638236/5156190
 try:
-    _dist = pkg_resources.get_distribution("cli50")
-    dist_loc = os.path.normcase(_dist.location)
-    here = os.path.normcase(__file__)
-    if not here.startswith(os.path.join(dist_loc, "cli50")):
-        raise pkg_resources.DistributionNotFound
+    __version__ = pkg_resources.get_distribution("cli50").version
```

Comment:
I _think_ @crossroads1112 was trying to handle this scenario, per https://stackoverflow.com/a/17638236/5156190:

> make this work in all cases, where you could end up running this without having installed it